tests/unit/lms/services/lti_outcomes_test.py

```diff
@@ -70,6 +70,15 @@ def test_record_result_sends_score(self, svc):
 
         assert score == "0.5"
 
+    @pytest.mark.usefixtures("response")
+    def test_record_result_sends_score_of_zero(self, svc):
+        svc.record_result(self.GRADING_ID, score=0)
+
+        result_record = self.sent_pox_body()["replaceResultRequest"]["resultRecord"]
+        score = result_record["result"]["resultScore"]["textString"]
+
+        assert score == "0"
```

Comment:
This could be combined into the previous test by making the previous test parameterized.